_studio/mfx_lib/encode_hw/h265/src/mfx_h265_encode_hw_utils.cpp

```diff
@@ -3586,7 +3606,8 @@ void ConfigureTask(
 
     //construct ref lists
     Zero(task.m_numRefActive);
```

Comment:
Why you did not change this Zero?

Reply:
I planned to get rid of Zero functions in later change